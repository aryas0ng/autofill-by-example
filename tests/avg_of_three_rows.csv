11,9.33333333333333
15,7.33333333333333
2,3.33333333333333
5,3.33333333333333
3,3
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,