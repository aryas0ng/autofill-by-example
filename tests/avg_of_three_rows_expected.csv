11,9.33333333333333
15,7.33333333333333
2,3.33333333333333
5,3.33333333333333
3,3
2,16.3333333333333
4,17.3333333333333
43,18
5,6.66666666666667
6,6
9,5.33333333333333
3,3
4,3.66666666666667
2,3
5,4.33333333333333
2,3
6,5
1,5
8,5
6,9.33333333333333
1,8.33333333333333
21,12
3,3